test/IonTextReader.ts

```diff
@@ -162,6 +162,19 @@ class IonTextReaderTests {
         assert.isNull(ionReader.next());
     }
 
+    @test "Parse through struct throws error on broken input"() {
+        let invalidIonToRead = "{broken";
+
+        let ionReader = ion.makeReader(invalidIonToRead);
+        ionReader.next();
+
```

Comment:
why wouldn't you use const instead of let since you don't modify or change the value of invalidIonToRead and ionReader

Reply:
Upgrading `let` to `const` where appropriate is something we intend to do project-wide via one of our linting tools but haven't had the opportunity yet. Once we do, new code will be required to `const` where possible.